tests/bootstrap.php

```diff
@@ -0,0 +1,56 @@
+<?php
+/**
+ * LICENSE
+ *
+ * This file is part of CFPropertyList.
+ *
+ * The PHP implementation of Apple's PropertyList can handle XML PropertyLists
+ * as well as binary PropertyLists. It offers functionality to easily convert
+ * data between worlds, e.g. recalculating timestamps from unix epoch to apple
+ * epoch and vice versa. A feature to automagically create (guess) the plist
+ * structure from a normal PHP data structure will help you dump your data to
+ * plist in no time.
+ *
+ * Copyright (c) 2018 Teclib'
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ * ------------------------------------------------------------------------------
+ * @author    Rodney Rehm <rodney.rehm@medialize.de>
+ * @author    Christian Kruse <cjk@wwwtech.de>
+ * @author    Thierry Bugier
```

Comment:
Aren't you going to add your email?

Reply:
I choosed to not put my email. 

One can contact me with the bug tracker of the repo (currently github) or via the company I work for. I like dark themes, I fear spam and  groupies (if this exists for developers...) :D 

https://github.com/flyve-mdm/glpi-plugin/blob/develop/inc/agent.class.php#L24

Reply:
@btry You can add the email of teclib <contact@teclib.com>